crates/rollup-boost/src/proxy.rs

```diff
@@ -172,7 +143,12 @@ where
 
                 // Fire and forget the builder request
                 tokio::spawn(async move {
-                    let _ = builder_client.forward(buffered_clone, method_clone).await;
+                    if let Err(err) = builder_client
+                        .forward(buffered_clone, method_clone.clone())
+                        .await
+                    {
+                        tracing::error!(method = %method_clone, error = ?err, "Builder forward request failed");
```

Comment:
I believe the .forward method already has an error log, though it might be worth adding the method in that error log (atm it only logs the http code)

Reply:
Good catch, I removed the redundant logging. The method should already be logged via the `forward` method.